Resources/Textures/Constructible/Misc/furniture.rsi/meta.json

```diff
@@ -104,6 +104,39 @@
     },
     {
       "name": "rack"
-    }
+    },
+	{
+	  "name": "rollerbed"
+	},
+	{
+	  "name": "rollerbed_buckled"
+	},
+	{
+	  "name": "rollerbed_folded"
+	},
+	{
+	  "name": "cheap_rollerbed",
+	  "copyright": "https://github.com/Baystation12/Baystation12"
+	},
```

Comment:
@SweptWasTaken probably knows more on split copyrights but I thought we dumped them in different rsis.

Reply:
I don't know what to do there, it would be pretty bad to have one rsi for each type of rollerbed

Reply:
Also I'm not sure if that is in the RSI spec.